paddle/fluid/inference/api/paddle_pass_builder.h

```diff
@@ -141,6 +142,9 @@ class GpuPassStrategy : public PassStrategy {
         "conv_elementwise_add_act_fuse_pass",        //
         "conv_elementwise_add2_act_fuse_pass",       //
         "conv_elementwise_add_fuse_pass",            //
+        // Disable cudnn gru fuse as model use different gru
```

Comment:
so these passes can't be used in the normally trained model?

Why not remove these three lines and call InsertPass to activate  them only in your test?

Reply:
updated, please help to review, thanks.